consumer/nf_management.go

```diff
@@ -123,3 +123,34 @@ func SendDeregisterNFInstance() (problemDetails *models.ProblemDetails, err erro
 	}
 	return
 }
+
+func SendUpdateNFInstance(patchItem []models.PatchItem) (nfProfile models.NfProfile, problemDetails *models.ProblemDetails, err error) {
+	logger.Consumerlog.Debugf("Send Update NFInstance")
+
+	pcfSelf := pcf_context.PCF_Self()
+	// Set client and set url
```

Comment:
Some of the comments not required. I suggest keep one or 2 liner description at the top of the function and that too only if function is very big.